commitlint.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
+};
```

Comment:
How do you feel about adding configs to `package.json`? I personally like it for small configs but I'm okay with it either way

```json
{
  "browserslist": [
    "> 0.1%",
    "not dead",
    "not op_mini all",
    "ie 11"
  ],
  "babel": {
    "presets": ["@babel/preset-env", "@babel/preset-react"],
    "plugins": ["@babel/plugin-proposal-class-properties"],
    "env": {
      "es": {
        "presets": [["@babel/preset-env", { "targets": { "esmodules": true } }]]
      }
    }
  },
  "commitlint": {
    "extends": ["@commitlint/config-conventional"],
  },
  "eslintConfig":{
    "parser": "babel-eslint",
    "env": {
      "browser": true,
      "jest": true
    },
    "extends": ["airbnb", "prettier"],
    "plugins": ["prettier"],
    "ignorePatterns": ["node_modules/", "dist/"],
    "rules": {
      "linebreak-style": 0,
      "react/jsx-filename-extension": 0,
      "react/destructuring-assignment": 0,
      "prettier/prettier": "error"
    }
  },
  "prettierConfig": {
    "printWidth": 120,
    "semi": true,
    "singleQuote": true,
    "trailingComma": "all"
  }
}
```

This config would replace `.babelrc`, `.eslintrc`, `.eslintignore`, `.prettierrc`, `browserslist` and `commitlint.config.js`
I do think it makes package.json become a bit unwieldy so maybe we still want a separate  `.babelrc` and `.eslintrc`?
Let me know what you think!

Reply:
I'm ok with either way. With the configs on its own files, we get autocomplete/linting on the code editor. It's just a bonus in my opinion.

Reply:
lets keep this as is for now!